webpack/ForemanTasks/Components/TasksTable/Components/ResumeConfirm.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```

Comment:
This file appears to be the same as `CancelConfirm.js`, just with different strings.  Why not consolidate them into a single file with values passed down as props?

Reply:
The ActionConfirmation is the general Modal that basically gets the strings and Cancel/Resume just gives it the correct props.
I also think having 2 different components its more readable instead of merging them into 1 component that has a lot of "ifs"